components/centraldashboard/app/server.ts

```diff
@@ -55,4 +55,9 @@ async function main() {
       () => console.info(`Server listening on port http://localhost:${port}`));
 }
 
+// This will allow us to inspect uncaught exceptions around the app
+process.on('unhandledRejection', error => {
+  console.log('[SEVERE] unhandledRejection', error);
```

Comment:
I'd log this as [error](https://nodejs.org/docs/latest-v10.x/api/console.html#console_console_error_data_args) just so that it can be surfaced more easily in tools like Stackdriver Logging.

Reply:
Makes sense, will do!